plugins/plugin-watermark/src/shared/options.ts

```diff
@@ -0,0 +1,42 @@
+import type { Page } from 'vuepress'
+import type { WatermarkOptions as _WatermarkRawOptions } from 'watermark-js-plus/dist/types/src/types/index.js'
+
+export type WatermarkOptions = Partial<_WatermarkRawOptions>
```

Comment:
This should be watermark client options, and I am.actually suggesting only provide complicated ones.

In most.cases we should expect users to set options in node.js side.